### PR TITLE
Carla: fix python package, move .pc file to -devel

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: srcpkgs/Carla/template [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Template file for 'Carla'
 pkgname=Carla
 version=1.9.8
-revision=1
+revision=2
 build_style=gnu-makefile
+pycompile_module="carla_utils.py carla_backend.py"
```
Comment (asergi) [inline srcpkgs/Carla/template]: Needs `pycompile_dirs="usr/share/carla"` too.

File: srcpkgs/Carla/template [hunk 2]
```diff
@@ -36,13 +37,15 @@ pre_build() {
...
+	vmkdir "$py3_sitelib"
+	mv ${DESTDIR}/usr/lib/python3/dist-packages/*.py "${DESTDIR}"/"$py3_sitelib"
 	vbin bin/carla-lv2-export
 }
 
 Carla-devel_package() {
 	short_desc+=" - development files"
 	pkg_install() {
 		vmove usr/bin/carla-lv2-export
+		vmove usr/lib/pkgconfig
```
Comment (asergi) [inline srcpkgs/Carla/template]: `vmove usr/include`

**Comment by maxice8** [comment]
On Wed, Jun 20, 2018 at 10:41:03AM -0700, Alessio Sergi wrote:
> asergi commented on this pull request.
> 
> 
> 
> >  build_style=gnu-makefile
> +pycompile_module="carla_utils.py carla_backend.py"
> 
> Needs `pycompile_dirs="usr/share/carla"` too.
> 
> 
> >  	vbin bin/carla-lv2-export
>  }
>  
>  Carla-devel_package() {
>  	short_desc+=" - development files"
>  	pkg_install() {
>  		vmove usr/bin/carla-lv2-export
> +		vmove usr/lib/pkgconfig
> 
> `vmove usr/include`
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/235#pullrequestreview-130515072

changed.

File: srcpkgs/Carla/template [hunk 3]
```diff
@@ -1,8 +1,10 @@
 # Template file for 'Carla'
 pkgname=Carla
 version=1.9.8
-revision=1
+revision=2
 build_style=gnu-makefile
+pycompile_module="carla_utils.py carla_backend.py"
+pycompile_dirs="/usr/share/carla"
```
Comment (asergi) [inline srcpkgs/Carla/template]: No absolute path in `pycompile_dirs` and you can merge at your convenience.

**Pull request closed without merging**